Code/GraphMol/Depictor/Wrap/testDepictor.py

```diff
@@ -616,90 +618,155 @@ def assertMolMatchesCoordMap(self, mol, coord_map):
-        rdDepictor.Compute2DCoords(two_linked_templates,useRingTemplates=False)
-        assert not self.molMatchesTemplate(two_linked_templates, template1)
-        assert not self.molMatchesTemplate(two_linked_templates, template2)
+        with tempfile.NamedTemporaryFile() as tmp_file:
+            # no coordinates
+            tmp_file.write(b"C1CCCCC1")
+            tmp_file.seek(0)
+            with self.assertRaises(ValueError):
+                rdDepictor.SetRingSystemTemplates(tmp_file.name)
 
-        rdDepictor.Compute2DCoords(two_linked_templates,useRingTemplates=True)
-        assert self.molMatchesTemplate(two_linked_templates, template1)
-        assert self.molMatchesTemplate(two_linked_templates, template2)
+        with tempfile.NamedTemporaryFile() as tmp_file:
+            # bridged ring system
+            tmp_file.write(b"c1ccccc1-c1ccccc1")
+            tmp_file.seek(0)
+            with self.assertRaises(ValueError):
+                rdDepictor.SetRingSystemTemplates(tmp_file.name)
 
-        # example with 3 templates linked together
-        three_linked_templates = Chem.MolFromSmiles("NC(CCC1CCC2C(C1)C1CC(CCC(=O)CC(N)CC~C3C4CC5CC3CC5C4)N2NN1)CC(=O)CCC1=CCCC2CCCCC2CCCCC2CCCC(CCCCCCC1)C2")
-        rdDepictor.Compute2DCoords(three_linked_templates,useRingTemplates=False)
-        assert not self.molMatchesTemplate(three_linked_templates, template1)
-        assert not self.molMatchesTemplate(three_linked_templates, template2)
-        assert not self.molMatchesTemplate(three_linked_templates, template3)
+        # set back to default ring system templates
+        rdDepictor.LoadDefaultRingSystemTemplates()
 
-        rdDepictor.Compute2DCoords(three_linked_templates,useRingTemplates=True)
-        assert self.molMatchesTemplate(three_linked_templates, template1)
-        assert self.molMatchesTemplate(three_linked_templates, template2)
-        assert self.molMatchesTemplate(three_linked_templates, template3)
+    def testAddRingSystemTemplates(self):
+        with rdDepictor.UsingCoordGen(False):
+            # this is what is in the default ring system templates
+            mol = Chem.MolFromSmiles("C1CCCN2CCCC(CC1)C2")
+            default_template = Chem.MolFromSmiles("C1CCCN2CCCC(CC1)C2 |(2.64,1.53,;3.1,0.11,;2.52,-1.28,;1.17,-1.94,;-0.28,-1.57,;-1.7,-2.03,;-2.82,-1.03,;-2.51,0.44,;-1.08,0.9,;-0.13,2.06,;1.35,2.31,;0.03,-0.1,)|")
+            new_template_smi = "C1CCCN2CCCC(CC1)C2 |(3.2585,-0.7733,;3.2585,0.7733,;2.225,1.9568,;0.6347,2.17,;-0.8068,1.4216,;-2.4372,1.3464,;-3.2585,0,;-2.4372,-1.3464,;-0.8068,-1.4216,;0.6347,-2.17,;2.225,-1.9569,;-0.6701,0,)|"
+            new_template = Chem.MolFromSmiles(new_template_smi)
 
-        rdDepictor.SetPreferCoordGen(prefer_coordgen_status)
+            rdDepictor.Compute2DCoords(mol, useRingTemplates=True)
+            assert self.molMatchesTemplate(mol, default_template)
 
+            with tempfile.NamedTemporaryFile() as tmp_file:
```

Comment:
This also needs:
```suggestion
            with tempfile.NamedTemporaryFile(delete=False) as tmp_file:
```
or the tests fail on Windows